R/params.R

```diff
@@ -250,6 +250,7 @@ params_html_head <- function(html_head_style = c(),
 #' You must take care to unsure that the URL is absolute.
 #' @param html_head_script_link same as above except that these are interpreted as SRC attributes in SCRIPT tags.
 #' You must take care to unsure that the URL is absolute.
+#' @param disable_bootstrap if true, does not inject boostrap scripts and styles in the HTML HEAD.
```

Comment:
I don't know what the common formatting is for this package, but you could consider styling with `if \code{TRUE}, does not...` Probably more important to align with the conventions of the package but just making sure you're aware of the option.